TransferMethod/Sources/AddTransferMethodSectionData.swift

```diff
@@ -50,8 +51,8 @@ final class AddTransferMethodSectionData {
     }
 
     init(fieldGroup: String,
-         country: String,
-         currency: String,
+         country: String? = nil,
```

Comment:
remove country and currency from fieldGroup: "INFORMATION" and fieldGroup: "CREATE_BUTTON" in AddTransferMethodController

Reply:
Removed country and currency